### DISCUSSION TITLE
这么好的框架，应该多推广推广，b站上都没有视频介绍。

### DISCUSSION THREAD

**Discussion by roc138** [shepherdpp/qteasy#143] [Show and tell]
我之前在淘宝（https://shop105517303.taobao.com）上买了一个tushare数据同步代码，核心代码还不开源，都要400元左右，回测框架套用backtrader要4000元左右。
你这个架构和文档说明做的真的非常棒。再稍微简化一点同步，同步所有数据，简化选择，让别人上手就能用，就可以卖钱了，大家也愿意买，毕竟可以节约很多时间，简化很多工作。

**Comment by shepherdpp** [comment]
感谢鼓励和支持！因为自己也要用，所以一切以好用为目标！
现在还没达到心目中完美的状态，1.2版要增加一个UI，还有内核代码还需要优化，不少地方需要完善。这些都在紧锣密鼓地进行中，希望有人能加入进来一起开发完善文档，能更快地迭代新版本。
